templates/request.pt.md

```diff
@@ -0,0 +1,27 @@
+Caro [REVISOR(A)]
+
+Olá, aqui é [EDITOR(A)]. [BRINCADEIRA AMIGÁVEL]. Estou escrevendo para perguntar se você gostaria de revisar um pacote para a rOpenSci. Como você provavelmente sabe, a rOpenSci realiza revisão por pares de pacotes de R contribuídos para nossa coleção, de maneira semelhante aos periódicos.
+
+O pacote, [PACOTE] de [AUTORES], faz [FUNÇÃO]. Você pode encontrá-lo no GitHub aqui: [LINK PARA REPOSITÓRIO]. Também conduzimos nosso processo de revisão aberta via GitHub, aqui: [ISSUE DE ONBOARDING]
+
+Se você aceitar, observe que pedimos aos revisores que concluam as avaliações em três semanas. (Descobrimos que a revisão de um pacote leva um tempo semelhante ao de um trabalho acadêmico.)
+
+Nosso [guia para revisores] detalha o que procuramos em uma revisão de pacote e inclui links para exemplos de revisão. Nossos padrões estão detalhados em nosso [guia de pacotes] e fornecemos um [modelo] de revisão para você usar. Certifique-se de que não haja um [conflito de interesses](https://devguide.ropensci.org/policies.html#coi) que o impeça de revisar este pacote. Se você tiver dúvidas ou comentários, sinta-se à vontade para me perguntar ou postar no [fórum da rOpenSci].
```

Comment:
"o impeça" assumes a male reader. To solve this, we can use "lhe" (too formal?) or "te" (technically incorrect without "impeças", but very common anyway). Or maybe adding more text like "... que seja uma impedimento para que possa revisar ..."

```suggestion
Nosso [guia para revisores] detalha o que procuramos em uma revisão de pacote e inclui links para exemplos de revisão. Nossos padrões estão detalhados em nosso [guia de pacotes] e fornecemos um [modelo] de revisão para você usar. Certifique-se de que não haja um [conflito de interesses](https://devguide.ropensci.org/policies.html#coi) que lhe impeça de revisar este pacote. Se você tiver dúvidas ou comentários, sinta-se à vontade para me perguntar ou postar no [fórum da rOpenSci].
```

Reply:
For me, "lhe" seems appropriate.